packages/contracts-bedrock/test/L2OutputOracle.t.sol

```diff
@@ -197,6 +197,30 @@ contract L2OutputOracle_getter_Test is L2OutputOracle_Initializer {
     }
 }
 
+contract L2OutputOracle_proposalManagement_Test is L2OutputOracle_Initializer {
+    /// @dev Tests that `isProposalManager` is identically enforced both on deletion and proposer updating
+    function testFuzz_isProposalManagerCanSetProposerAndDelete_succeeds(address caller) external {
+        proposeAnotherOutput();
+
+        uint256 latestOutputIndex = oracle.latestOutputIndex();
+        address newProposer = makeAddr("New Proposer");
+
+        if (sysConf.isProposalManager(caller)) {
+            vm.prank(caller);
+            sysConf.setProposer(newProposer);
+            oracle.deleteL2Outputs(latestOutputIndex);
```

Comment:
Do you need to prank as `newProposer` before calling `deleteL2Outputs` here? Isn't that supposed to be gated so it should revert?

Reply:
No, the Proposer is different from the Challenger. Pranking as the proposer should cause a revert.
This test is just meant to indicate that the abilities to
1. delete outputs, and 
2. update the Proposer address,
are shared by the same set of actors.

I'm starting to think that `isProposalManger()` isn't the best name, but I can't think of a better one. Nor can chatGPT.

But I'm adding this comment to that function to hopefully clarify:
```
///         The same entities who can delete an output should also be able to update the proposer; because in the
///         event that a faulty output is proposed, the malicious proposer will need to be removed prior to
///         deleting the output.
```